api/v4/source/system.yaml

```diff
@@ -1125,6 +1125,25 @@
         ##### License
         Requires either a E10 or E20 license.
       operationId: GenerateSupportPacket
+      parameters:
+        - name: basic.server.logs
```

Comment:
It's a bit odd to use dots in a query parameter. Other API endpoints use an underscore. Is there a specific reason that dots are used for this one?

Reply:
No specific reason TBH, it's actually -once thought with plugins- looks like below:
```
- basic.server.logs=true
- plugin_packets={com.mattermost.mattermost-plugin-metrics}
```
But I'm okay with underscores as well. Let me know if you think an update is necessary. cc @hanzei